android/app/src/main/res/layout/item_category_description.xml

```diff
@@ -45,6 +46,7 @@
         android:padding="@dimen/margin_base"
         android:text="@string/category_desc_more"
         android:textColor="?attr/colorAccent"
+        android:visibility="gone"
```

Comment:
```suggestion
        
```

I don't think this line changes anything, the visibility is always set programmatically.

Reply:
can I set the visibility through programmatically .

Reply:
You don't need to do anything here, just remove the line. The visibility is set programmatically / from the Java code in BookmarkListAdapter.java.

Reply:
Done